etc/listr2.api.md

```diff
@@ -208,7 +208,7 @@ export class ListrTaskObject<Ctx, Renderer extends ListrRendererFactory> extends
     hasSubtasks(): boolean;
     hasTitle(): boolean;
     id: string;
-    initialTitle?: string;
+    initialTitle?: string | null;

```

Comment:
backwards compatible because: without `strictNullChecks: true`, this could have been `null` all along